int32 index
int32 addr
int32 count
string valTab
---
int32 res